### ISSUE TITLE
JINJA logic evaluation trims off spacing

### ISSUE THREAD

**Issue by drpatelh** [nf-core/tools#1826]
Due to the code [here](https://github.com/nf-core/tools/blob/bc56cdd35077e408a0a7c90ec07fcdff7c2d1f3b/nf_core/pipeline-template/main.nf#L7-L9) when the pipeline is actually created we lose the leading whitespace in front of `Website` as you can see [here](https://github.com/nf-core/rnaseq/blob/b7baabfe1adfb99df96bd31475e36388d5694613/main.nf#L7-L8).

**Comment by ewels** [comment]
Easy fix: `{% if branded -%}` should become `{% if branded %}` (NB no `-` at the end).

May get a linting failure from Prettier after that. If so, need to move it onto the same line as the `website`

eg:
```
{% if branded %}    Website: https://nf-co.re/{{ short_name }}
    Slack  : https://nfcore.slack.com/channels/{{ short_name }}
```

**Comment by maxulysse** [comment]
Same issue in the README:
(there's a space missing towards the end of the line here: https://github.com/nf-core/isoseq/blob/dev/README.md?plain=1#L29)

**Comment by ewels** [comment]
@maxulysse I don't understand what you mean here.

This line in the template is vanilla markdown without any Jinja tags:
https://github.com/nf-core/tools/blob/bc56cdd35077e408a0a7c90ec07fcdff7c2d1f3b/nf_core/pipeline-template/README.md?plain=1#L32

**Comment by mirpedrol** [comment]
I will close it as it is fixed by #1830, but feel free to reopen if there are still issues.
